moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -139,7 +139,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     attached_object_publisher_ = node_handle_.advertise<moveit_msgs::AttachedCollisionObject>(
         planning_scene_monitor::PlanningSceneMonitor::DEFAULT_ATTACHED_COLLISION_OBJECT_TOPIC, 1, false);
 
-    current_state_monitor_ = getSharedStateMonitor(robot_model_, tf_, node_handle_);
+    current_state_monitor_ = getSharedStateMonitor(opt_.robot_description_, robot_model_, tf_, node_handle_);
```

Comment:
i don't understand why this change is necessary - this slows down the code by requiring a string to be parsed rather than using the pre-constructed RobotModel class